settings.gradle.kts

```diff
@@ -17,14 +17,14 @@ include("studio-agent")
 include("studio-plugin")
 
 rootProject.children.forEach {
-    it.projectDir = rootDir.resolve( "subprojects/${it.name}")
+    it.projectDir = rootDir.resolve("subprojects/${it.name}")
 }
 
 /**
- * Intellij-gradle-plugin requires Java 11.
+ * Multiple modules, including Gradle Profiler require Java 17 toolchain
  */
 fun checkIfCurrentJavaIsCompatible() {
```

Comment:
Could this be removed and use [Daemon Toolchains](https://docs.gradle.org/current/userguide/gradle_daemon.html#sec:configuring_daemon_jvm) instead?

If so, please also add:
```kotlin
tasks.named<UpdateDaemonJvm>("updateDaemonJvm") {
    toolchainDownloadUrls.empty()
}
```